backends/arm/quantizer/arm_quantizer.py

```diff
@@ -218,9 +218,35 @@ def not_module_type_or_name_filter(n: Node) -> bool:
 
 class TOSAQuantizer(Quantizer):
 
-    def __init__(self, tosa_spec: TosaSpecification) -> None:
+    def __init__(
+        self, compile_spec_or_tosa_spec: Union[TosaSpecification, List[CompileSpec]]
```

Comment:
Nit: do you want to split into two constructors, instead of overloading..
```
def __init__(self, tosa_spec):
[...]
```
and
```
@classmethod
def from_compilespec(self, compilespec):
[...]
```